src/components/Authentication/Signin.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+
+// Assets
```

Comment:
Remove this comment as it doesn't add value to the code.

Reply:
> Hey @ZhraaBaharAluloom nice work here. Can you please see some comments I left you and fix the deployment error. Thank you!

Added the changes